### PR TITLE
TW-1196 Add collection template links to LC

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/sending-requests/authorization/authorization.md [hunk 1]
```diff
@@ -42,3 +42,5 @@ You can pass auth details along with any request you send in Postman. Auth data
 * If you have session cookies in your browser, you can sync them to Postman using the Interceptor. See [Interceptor extension](/docs/sending-requests/capturing-request-data/interceptor/) and [Cookies](/docs/sending-requests/cookies/) for more detail.
 * If you're having issues getting a request to authenticate and run successfully, review the tips in [troubleshooting API requests](/docs/sending-requests/troubleshooting-api-requests/). If you still have auth problems, check out the [authentication tag](https://community.postman.com/tags/authentication) on the Postman forum.
 * For more details on each authorization type, visit [About authorization types](/docs/sending-requests/authorization/authorization-types/).
+
+> <img alt="Collections icon" src="https://assets.postman.com/postman-docs/Collections.png#icon" width="24px"> Try out examples of different types of authorization in a collection template that are ready to be modified to fit your use case. To try out this template, select [Authorization methods](https://postman.postman.co/workspace/6311738d-2e70-441f-ae12-78caf078c5b7/collection/24034743-31a9a6ed-4cdf-4ced-984c-d12c9aec1c27?action=share&creator=24552533).
```
Comment (mark-anthony-postman) [inline src/pages/docs/sending-requests/authorization/authorization.md:46]:
```suggestion
> <img alt="Collections icon" src="https://assets.postman.com/postman-docs/Collections.png#icon" width="24px"> Try out examples of different types of authorization in a collection template that's ready to be modified to fit your use case. To try out this template, select [Authorization methods](https://postman.postman.co/workspace/6311738d-2e70-441f-ae12-78caf078c5b7/collection/24034743-31a9a6ed-4cdf-4ced-984c-d12c9aec1c27?action=share&creator=24552533).
```

**Review by mark-anthony-postman — APPROVED** [review]
LGTM

File: src/pages/docs/collections/running-collections/intro-to-collection-runs.md [hunk 2]
```diff
@@ -180,3 +180,5 @@ In addition to running collections manually, the Collection Runner enables you t
 Use the Collection Runner fundamentals you've learned to extend its functionality with workflows.
 
 * To learn how to use scripts to build workflows with conditional sequences for running the requests in your collections, visit [Building request workflows](/docs/collections/running-collections/building-workflows/).
+
+> <img alt="Collections icon" src="https://assets.postman.com/postman-docs/Collections.png#icon" width="24px"> See how an integration test can ensure that all individual components of an API function together seamlessly. To try out this template, select [Integration testing](https://postman.postman.co/workspace/6311738d-2e70-441f-ae12-78caf078c5b7/collection/22517504-fe506090-ca91-4340-bea9-82d2c3d2bb9a?action=share&creator=24552533).
```
[the remark this answers is not in the document]
Comment (jkonrath-postman) [reply src/pages/docs/collections/running-collections/intro-to-collection-runs.md]: No need to fork with the new links! I change this to reflect that.

**Pull request merged**